Bobsar0/Fast-Food-Fast
# #160840091 Setup PostgreSQL database

#### What does this PR do?
Sets up local and test databases using node-postgres (pg)

#### Description of Task to be completed?
- Setup connection to local PostgreSQL database using connection-string hosted on .env file
- Setup connection to travis PostgreSQL database for testing purposes.
- Create Users table
- Create Orders table with relation to Users table

#### How should this be manually tested?
Clone the repo and run the following commands:
```
  git checkout ch-setup-database-#160840091
  npm test
  npm start
```
You should see an output on the console indicating success in creation of tables

What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/#160840091
